refinery/tool_manager/tests.py

```diff
@@ -309,6 +308,7 @@ def create_solr_mock_response(self, nodes):
             }
         )
 
+    @override_settings(CELERY_ALWAYS_EAGER=True)
```

Comment:
It makes me worry a little if the tests depend on Celery...? Particularly since the actual run seems to be mocked.

Reply:
This setting runs Celery on a single thread so that we don't have to worry about sleeping/waiting after launching a Tool. I could probably mock the underlying `delay()` call but this seemed easier.